redis/variables.tf

```diff
@@ -65,3 +65,8 @@ variable "snapshot_retention_limit" {
   description = "The number of days for which ElastiCache will retain automatic cache cluster snapshots before deleting them. For example, if you set SnapshotRetentionLimit to 5, then a snapshot that was taken today will be retained for 5 days before being deleted. If the value of SnapshotRetentionLimit is set to zero (0), backups are turned off. Please note that setting a snapshot_retention_limit is not supported on cache.t1.micro or cache.t2.* cache nodes"
   default     = "0"
 }
+
+variable "snapshot_arns" {
```

Comment:
shouldn't this be `snapshot_arn` instead of `snapshot_arns`?

Reply:
I used the same name as in the terraform provider. btw I fixed the type of the parameter, as it should be a list

Reply:
Can you also fix the description and documentation? It refers to a single element string with an example of a string.

Reply:
Never mind, wording was confusing but correct :)